apps/dashboard/config/configuration_singleton.rb

```diff
@@ -330,6 +330,22 @@ def file_upload_max
     [ENV['FILE_UPLOAD_MAX']&.to_i, ENV['NGINX_FILE_UPLOAD_MAX']&.to_i].compact.min || 10737420000
   end
 
+  # The timeout (seconds) for "generating" a .zip from a directory.
+  #
+  # Default for OOD_DOWNLOAD_DIR_TIMEOUT is "5" (seconds).
+  # @return [Integer]
+  def file_download_dir_timeout
+    ENV['OOD_DOWNLOAD_DIR_TIMEOUT']&.to_i || 5
```

Comment:
OOD_DOWNLOAD_DIR_TIMEOUT_SECONDS might be better to remind someone what the unit is?